knot-thing/src/main/java/com/cesar/knot_sdk/knot_state_machine/states/base_classes/State.kt

```diff
@@ -0,0 +1,34 @@
+package com.cesar.knot_sdk.knot_state_machine.states.base_classes
+
+abstract class State {
+
+    val NO_ERROR_MESSAGE : String = "no error"
+    val NETWORK_ERROR_MESSAGE : String = "no error"
+    val TIMEOUT_MESSAGE  : String = "TIMEOUT"
+    val SUCCESS_MESSAGE : String = "Success"
+    var error : String = NO_ERROR_MESSAGE
```

Comment:
Use a pattern for the values of the constants. Write the strings on lowercase, uppercase or the first letter in uppercase, maybe you can keep the pattern used on the next commit. Also, I think you can use the `'const'` keyword on these values.